src/Orchard.Web/Modules/Orchard.Modules/Controllers/AdminController.cs

```diff
@@ -132,6 +135,20 @@ public class AdminController : Controller {
                 .ToList();
             }
 
+            if (!string.IsNullOrEmpty(executionId))
+            {
+                var result = _recipeResultAccessor.GetResult(executionId);
+
+                if (result.IsSuccessful)
+                {
+                    Services.Notifier.Information(T("The recipe was executed successfully. Check the logs (recipe execution ID <strong>{0}</strong>) for more information.", result.ExecutionId));
```

Comment:
Would it be useful to redirect to separate result page like on Orchard.ImportExport module when import is executed? That way it would be more convienent to show details of recipe execution results both on success and failure.